contrib/docker/bitcon.conf

```diff
@@ -0,0 +1,692 @@
+##
+## bitcoin.conf configuration file.
+## Generated by contrib/devtools/gen-bitcoin-conf.sh.
```

Comment:
Storing a duplicate bitcoin.conf (when one can be generated with `get-bitcoin-conf.sh` directly from `bitcoind --help`) would lead to increased maintenance effort.

Reply:
Thanks for your replies, I learned a lot. This is my first bitcoin core path, I will continue to learn